packages/dom/test/serialize-dom.test.js

```diff
@@ -68,6 +78,89 @@ describe('serializeDOM', () => {
       expect(html).not.toMatch('<template shadowroot');
       expect(html).not.toMatch('Hey Percy!');
     });
+
+    it('renders single nested ', () => {
+      if (!navigator.userAgent.toLowerCase().includes('chrome')) {
+        return;
+      }
+
+      withExample('<div id="content"></div>', { withShadow: false });
+      const baseContent = document.querySelector('#content');
+
+      const el1 = createShadowEl(1);
+      const el2 = createShadowEl(2);
+      el1.shadowRoot.appendChild(el2);
+      baseContent.append(el1);
+
+      const html = serializeDOM().html;
+
+      expect(html).toMatch(new RegExp([
+        '<template shadowroot="open">',
+        '<p>Percy-1</p>',
+        '<div id="Percy-2" .*>',
+        '<template shadowroot="open">',
+        '<p>Percy-2</p>',
+        '</template>'
+      ].join('')));
+    });
+
+    it('renders many nested', () => {
+      if (!navigator.userAgent.toLowerCase().includes('chrome')) {
+        return;
+      }
+      withExample('<div id="content"></div>', { withShadow: false });
+      const baseContent = document.querySelector('#content');
+
+      const levels = 5;
```

Comment:
```suggestion
      const levels = 10000;
```

let's use the biggest threshold.

Reply:
Using 1000